desktop/linux/space.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Disk utilization
+keywords: linux, disk
+title: Disk utilization in Docker for Linux
+redirect_from:
+- /linux/space/
+---
+
+Docker Desktop stores Linux containers and images in a single, large "disk image" file in the Linux filesystem. This is different from Docker on Linux, which usually stores containers and images in the `/var/lib/docker` directory.
+
+## Where is the disk image file?
+
+To locate the disk image file, select the Docker icon and then
+**Settings** > **Resources** > **Advanced**.
+
+![Disk preferences](images/menu/prefs-advanced.png){:width="750px"}
+
+The **Advanced** tab displays the location of the disk image. It also displays the maximum size of the disk image and the actual space the disk image is consuming. Note that other tools might display space usage of the file in terms of the maximum file size, and not the actual file size.
+
+## If the file is too big
```

Comment:
(Not a blocker, just "thinking out loud") I think this was copied from the "for mac" content; wondering is "big" correct, or should this be "too large"?

Perhaps one of the native english people knows